index.html

```diff
@@ -101,22 +88,23 @@ <h2 class="editable">Who?</h2>
 		<div class="container">
 			<div class="text">
 				<h2 class="editable">Where?</h2>
-				<p class="editable">Facebook, Paris</p>
+				<p class="editable">EuroScipy, Basel, Switzerland</p>
+				<p class="editable">Paris-Saclay, France</p>
 			</div>
 		</div>
 		<div class="container">
 			<div class="text">
 				<h2 class="editable">When?</h2>
-				<p class="editable">The sprint will take place Monday April 22<sup>nd</sup> - Friday April 26<sup>th</sup>, 2019. The day-by-day schedule is TBD.</p>
+				<p class="editable">The sprint will start in August 18th, 2023 and will have an end moment in <September></September> 5-7th. You can contribute both virtually and in person..</p>
```

Comment:
huh @bruAristimunha why 5-7, should be 4-8 no?

Reply:
No strong opinions, what do you think about the date @tomMoral? I think you have better experience about the days and the infra in paris-saclay side.

Reply:
I think 3 days are usually enough, in particular, as you plan to work in between the 2 events. More than that tends to make people less committed to the sprint as it is hard to get more free days to code :laughing: 

On my side, I will only be able to join on the 6th so I have a preference if it is included in the 3 days :)

Reply:
I am available to join as many days as needed. The point is that I think that part of the @tomMoral/@agramfort team will only be fully available on the 6th. We will need Thomas for rooms and infrastructure. Maybe we can have an internal meeting on the 5th, and try to arrange everything to make four days, with 3 for the community. What do you think?